ruby/BUILD.bazel

```diff
@@ -161,6 +162,29 @@ conformance_test(
     }),
 )
 
+genrule(
+    name = "copy_ruby_amalgamation_h",
+    srcs = ["@upb//:ruby-upb.h"],
+    outs = ["generated-in/ext/google/protobuf_c/ruby-upb.h"],
+    cmd = "cp $< $@",
+)
+
+genrule(
+    name = "copy_ruby_amalgamation_c",
+    srcs = ["@upb//:ruby-upb.c"],
+    outs = ["generated-in/ext/google/protobuf_c/ruby-upb.c"],
```

Comment:
It might be nice to allow the staleness test to more compactly support this case (i.e. just pass it the @upb targets).  These genrules seem a bit hacky